LabTools/CoreWidgets/ScriptWidget.py

```diff
@@ -13,40 +12,62 @@
 if USE_PYQT5:
 
     from PyQt5.QtWidgets import (QWidget, QLabel, QLineEdit, QPushButton,
-                                 QFileDialog, QHBoxLayout, QApplication)
+                                 QFileDialog, QHBoxLayout, QApplication, QComboBox, QSizePolicy)
 
 else:
 
     from PyQt4.QtGui import (QWidget, QLabel, QLineEdit, QPushButton,
-                             QFileDialog, QHBoxLayout, QApplication)
+                             QFileDialog, QHBoxLayout, QApplication, QComboBox, QSizePolicy)
+
+try:
+    from pathlib import Path
+    PATHLIB=True
+except:
+    PATHLIB=False
 
 
 from LabTools.IO import IOTool
+import os
 
+LEGACY_SCRIPTWIDGET = False
 
 class ScriptWidget(QWidget):
-    """widget with a lineEdit and a browse button"""
+    """widget with a dropdown and a browse button"""
 
-    def __init__(self, parent=None):
+    def __init__(self, parent=None, LEGACY = None):
```

Comment:
```suggestion
    def __init__(self, parent=None, legacy=None):
```
function arguments should be small caps